tdwii_plus_examples/ngetscu.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import sys
+from typing import Iterator, List
+
+import pydicom
+import pynetdicom
+from pydicom import DataElement, Dataset
+from pynetdicom import UnifiedProcedurePresentationContexts
+from pynetdicom.sop_class import UnifiedProcedureStepPush
+
+import tdwii_plus_examples.tdwii_config as ae_config
+
+
+def nget_ups(
+    attribute_list: List[DataElement], ups_instance_uid: str, scu_ae_title: str, scp_ae_title: str
+) -> Iterator[tuple[Dataset, Dataset | None]]:
+    """Perform UPS N-GET-RQ and return responses that have UPS content
+
+    Args:
+        ds_query (pydicom.Dataset): query content/request
```

Comment:
ds_query not in the args. attribute_list and ups_instance_uid missing from this docstring

Reply:
fixed in 0e0d99882130b7aee1bd5f0ae0b6df0193e7b4e2